src/into_dart.rs

```diff
@@ -92,43 +109,89 @@ impl IntoDart for CString {
     }
 }
 
-trait Num8Bit {}
+impl IntoDartExceptPrimitive for CString {}
 
-impl Num8Bit for u8 {}
+pub(crate) trait DartTypedDataTypeVisitor {
+    fn visit<T: DartTypedDataTypeTrait>(&self);
+}
```

Comment:
Could you please explain (adding more comments to the code) about this trait? like why do we need it?